### ISSUE TITLE
Values out of the range of breaks are white, but could be more informative if saturated

### ISSUE THREAD

**Issue by SamGG** [alexploner/Heatplus#7]
Hi,
My wish is that values higher (resp. lower) than the maximum (resp. minimum) of the breaks argument should be represented with the top (resp. bottom) color of the color palette. Currently, ceiling (resp. floor) had to be done before heatmap.
Best.

**Comment by alexploner** [comment]
Thanks for the feedback, but I disagree: values outside of the range of breakpoints should only occurr if the user has specified their own set of breakpoints - the default should cover the whole range. In the situation where the user-specified breaks fail to do so, this may or may not be intentional; I strongly prefer giving the user direct visual feedback on the consequences of their input, rather than guessing what they may have meant, and producing a potentially misleading figure.

**Comment by SamGG** [comment]
Thanks for your answer. Right, I tuned the breaks. I understand your point of view. My view is different because I refer to a very few and very extreme outliers.  I consider that those points should be saturated. I agree that it's fine to inform the user about those. But as I tun the breaks (knowing that outliers are present), truncating outliers sounds to me the way to directly get a meaningful view. The current behaviour is sometimes misleading because those outliers are not easily identified: they are white, ie the same colour as the mid colour of a blue-white-red scale (my favourite). May be it's not an issue with a green-black-red palette. I am not intending to change your choice. I am just bringing you another point of view. May be a caution in the doc should highlight this. Best.

**Comment by alexploner** [comment]
I agree that with the current behavior, you will generally not be able to spot a small number of outliers, as a few white pixels will not stand out in the plot. However, in practice I would not be much concerned - the point of the heatmap is to show global structure rather than finding rare deviations from it. And of course, truncating the range without some kind of warning will _not_ make it easier to spot a few or even a large number of outliers. 

I will leave the issue open for now and think about a better solution (though with low priority). THanks for the feedback!

**Comment by SamGG** [comment]
Final decision returns to you, the developer, you.
I appreciate your interest.
Cheers.
